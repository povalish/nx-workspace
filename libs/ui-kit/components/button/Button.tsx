import { IButton } from "./Button.interface";

//
//

export const Button: React.FC<IButton> = ({ children, onPress }) => (
  <button type='button' onClick={onPress}>{children}</button>
)