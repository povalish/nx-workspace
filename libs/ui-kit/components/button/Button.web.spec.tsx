import React from 'react';
import { render } from '@testing-library/react';

import { Button } from './Button';

describe('Button component', () => {
  it('should render successfully', () => {
    const { baseElement } = render(<Button>Click me!</Button>);
    expect(baseElement).toMatchSnapshot();
  });
});
