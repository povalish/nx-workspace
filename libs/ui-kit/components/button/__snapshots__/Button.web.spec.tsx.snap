// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component should render successfully 1`] = `
<body>
  <div>
    <button
      type="button"
    >
      Click me!
    </button>
  </div>
</body>
`;
